package.json

```diff
@@ -3,12 +3,18 @@
   "version": "2.1.0",
   "description": "A DAISY Online Delivery Protocol compatible book player for handheld devices",
   "private": true,
+  "scripts": {
+    "watch": "npx coffee ./server.coffee",
+    "build:dev": "npx cake -d app",
+    "build": "npx cake -cmt app"
+  },
   "directories": {
     "doc": "doc"
   },
   "devDependencies": {
     "body-parser": "^1.8.4",
     "coffeelint": "^2.1.0",
+    "coffeescript": "^1.11.1",
```

Comment:
should we lock this? Alternatively we could check-in package-lock.json, so that nota.dk can `npm ci` in their build script